queries/futures/useGetFuturesOpenOrders.ts

```diff
@@ -9,61 +9,91 @@ import { ETH_UNIT } from 'constants/network';
 import QUERY_KEYS from 'constants/queryKeys';
 import Connector from 'containers/Connector';
 import { futuresAccountState, openOrdersState, marketInfoState } from 'store/futures';
+import { formatCurrency } from 'utils/formatters/number';
+import { FuturesMarketAsset, getMarketName, MarketKeyByAsset } from 'utils/futures';
 import logError from 'utils/logError';
 
+import { PositionSide, FuturesOrder } from './types';
 import { getFuturesEndpoint } from './utils';
 
 const useGetFuturesOpenOrders = (options?: UseQueryOptions<any>) => {
 	const { selectedFuturesAddress } = useRecoilValue(futuresAccountState);
-	const { network } = Connector.useContainer();
+	const { network, synthsMap } = Connector.useContainer();
 	const futuresEndpoint = getFuturesEndpoint(network?.id as NetworkId);
 
 	const marketInfo = useRecoilValue(marketInfoState);
 	const setOpenOrders = useSetRecoilState(openOrdersState);
 
-	return useQuery<any[]>(
+	return useQuery<FuturesOrder[]>(
 		QUERY_KEYS.Futures.OpenOrders(network?.id as NetworkId, selectedFuturesAddress),
 		async () => {
 			if (!selectedFuturesAddress) {
 				setOpenOrders([]);
 				return [];
 			}
 			try {
-				const marketAddress = marketInfo?.market;
+				const marketAsset = marketInfo?.assetHex;
 				const response = await request(
 					futuresEndpoint,
 					gql`
-						query OpenOrders($account: String!, $market: String!) {
-							futuresOrders(where: { account: $account, market: $market, status: Pending }) {
+						query OpenOrders($account: String!, $asset: String!) {
+							futuresOrders(where: { abstractAccount: $account, asset: $asset, status: Pending }) {
 								id
 								account
 								size
 								market
 								asset
 								targetRoundId
+								targetPrice
 								timestamp
 								orderType
 							}
 						}
 					`,
-					{ account: selectedFuturesAddress, market: marketAddress }
+					{ account: selectedFuturesAddress, asset: marketAsset }
 				);
 
-				const openOrders = response
-					? response.futuresOrders.map((o: any) => ({
-							...o,
-							asset: ethersUtils.parseBytes32String(o.asset),
-							targetRoundId: new Wei(o.targetRoundId, 0),
-							size: new Wei(o.size).div(ETH_UNIT),
-					  }))
+				const openOrders: FuturesOrder[] = response
+					? response.futuresOrders.map((o: FuturesOrder) => {
+							const asset: FuturesMarketAsset = ethersUtils.parseBytes32String(
+								o.asset
+							) as FuturesMarketAsset;
+							const size = new Wei(o.size).div(ETH_UNIT);
+							const targetPrice = new Wei(o.targetPrice).div(ETH_UNIT);
```

Comment:
I'm not sure it's best naming but I added a util for this: `weiFromWei` in number utils.